README.md

```diff
@@ -103,41 +111,88 @@ setSubjectData({ // All parameters optional
 trackSelfDescribingEvent({ // Self-describing JSON for the custom event
   'schema': 'iglu:com.acme/event/jsonschema/1-0-0',
   'data': {'message': 'hello world'}
-  },
-  []); // array of self-describing JSONs for custom contexts, or an empty array if none are to be attached
+});
 ```
 
 ### Track a structured event:
 
 ```JavaScript
-trackStructuredEvent('my-category', // required, category, string
-                    'my-action', // required, action, string
-                    'my-label', // optional - set to null if not needed, label, string
-                    'my-property', // optional - set to null if not needed, property, string
-                    50.00, // optional - set to null if not needed, value, number
-                    []); // array of self-describing JSONs for custom contexts, or an empty array if none are to be attached
+trackStructuredEvent({
+  // required
+  category: 'my-category',
+  action: 'my-action',
+  // optional
+  label: 'my-label',
+  property: 'my-property',
+  value: 50.00
+  }
+);
 ```
 
 ### Track a Screen View:
 
 Note - for the track Screen View method, if previous screen values aren't set, they should be automatically set by the tracker.
 
 ```JavaScript
-trackScreenViewEvent('my-screen-name', // required, screen name, string
-                    '63ddebea-a948-4e0c-b458-96467d46f230', // optional - set to null if not needed (recommended), screen id, /uuid4 string
-                    'my-screen-type' // optional - set to null if not needed (recommended), screen type, string
-                    'my-previous-screen', // optional - set to null if not needed (recommended), previous screen name, string
-                    'my-previous-screen-type', // optional - set to null if not needed (recommended), previous screen type, string
-                    'e4711a72-c721-4dfa-b51a-a2e201dcec09', // optional - set to null if not needed (recommended), previous screen id, UUID4 string
-                    'my-transition-type', // optional - set to null if not needed (recommended), transition type, string
-                    []); // array of self-describing JSONs for custom contexts, or an empty array if none are to be attached
+trackScreenViewEvent({
+  screenName: 'my-screen-name', //required
+  // optional:
+  screenType: 'my-screen-type',
+  transitionType: 'my-transition'
+  }
+);
 ```
 
 ### Track a Page View:
 
 ```JavaScript
-trackPageViewEvent('my-page-url.com', // required, page url, string
-                  'my page title', // optional - set to null if not needed, page title, string
-                  'my-page-referrer.com', // optional - set to null if not needed, referrer url, string
-                  []); // array of self-describing JSONs for custom contexts, or an empty array if none are to be attached
+Tracker.trackPageViewEvent({
```

Comment:
This example is different to all the rest. It's `Tracker.track...` whereas all the other examples are `track...`